Plugins/Encounter/NWScript/nwnx_encounter.nss

```diff
@@ -53,6 +53,16 @@ int NWNX_Encounter_GetPlayerTriggeredOnly(object encounter);
 /// @param playerTriggeredOnly TRUE/FALSE
 void NWNX_Encounter_SetPlayerTriggeredOnly(object encounter, int playerTriggeredOnly);
 
+/// @brief Get if the encounter respawns or not.
+/// @param encounter The encounter object.
+/// @return TRUE if the encounter does respawn, FALSE otherwise.
+int NWNX_Encounter_GetReset(object encounter);
```

Comment:
The function name is not very descriptive.. maybe `{G,S}etRespawning

Reply:
I thought about that too but at the same time that just seems to be it's name.
The member variable is called reset and so is the field in the gff. There are also the two related functions GetResetTime, SetResetTime.
People who are familiar with the gff might look/search for something with reset.

Should I change it anyway?

Reply:
For what it's worth, the internal name is actually `{G,S}etEncounterCanReset()`, it's just the functions are not exposed in the API, only the field. I think something GetReset() is too confusing. GetCanReset() would be much better.

Reply:
Changed it to GetCanReset, SetCanReset.